### ISSUE TITLE
Line in unexpected format

### ISSUE THREAD

**Issue by scottwthompson** [TeamHG-Memex/eli5#364]
Python version 3.7.3
xgboost version 0.82

```
    input = pd.DataFrame([datum])
    prediction_html = show_prediction(
       xgb_model,
        input,
        feature_names=feature_names,
        targets=[True],
        show_feature_values=True
    )

>>> 

ValueError: Line in unexpected format: 		5:[has_transacted] yes=11,no=12,gain=336.531525,cover=32763.1055
```

The actual model predict with the same inputs. Any advice? is xgboost 0.82 supported?

**Comment by scottwthompson** [comment]
Also, I was getting this library to work just fine with the exact same code. But then I changed my model artifact to include this boolean feature and it stopped working.

So don't think its an issue with my code issue so much.

**Comment by giffordjoe** [comment]
I believe the issue is having booleans in the input.  when i casted my boolean columns to integers everything works as normal.

My guess is XGBoost is doing this conversion under the hood and the eli5 library is not.
